packages/create-app/src/createApp.ts

```diff
@@ -77,7 +77,7 @@ async function buildApp(appDir: string) {
 
 async function moveApp(tempDir: string, destination: string, id: string) {
   await Task.forItem('moving', id, async () => {
-    await fs.move(tempDir, destination).catch(error => {
+    await fs.move(tempDir, destination, { overwrite: true }).catch(error => {
```

Comment:
Why do we need this?

Reply:
This was required to `move` the temporary directory to a directory that already exists. I can see there being concern with this overwriting a directory that contains files/folders.

Reply:
yep - think we should probably skip this, and make sure it's a clean directory everytime.

Reply:
@benjaminr curious what you think of the latest commit (1d7078d). I now validate that the path option is a directory and exists, and no longer rely on the `overwrite` option, which was a pretty dangerous way of doing things!

To get around the `overwrite` option, I had to update the `moveApp` function to recursively copy files, instead of moving the temporary directory to the new directory. This was done to preserve files in the directory, which is required for @OrkoHunter's original use-case of creating an application in a Git repository where `.git/` files already exist.

Thanks again for all of the feedback!